api/v1beta1/types.go

```diff
@@ -635,6 +640,48 @@ type ServiceEndpointSpec struct {
 	Locations []string `json:"locations"`
 }
 
+// PrivateLinkServiceConnection defines the specification for a private link service connection associated with a private endpoint.
+type PrivateLinkServiceConnection struct {
+	// Name specifies the name of the private link service.
+	// +optional
+	Name string `json:"name,omitempty"`
+	// PrivateLinkServiceID specifies the resource ID of the private link service.
+	PrivateLinkServiceID string `json:"privateLinkServiceID"`
```

Comment:
Ditto like the other fields:
```suggestion
	PrivateLinkServiceID string `json:"privateLinkServiceID,omitempty"`
```
I suppose this isn't strictly necessary since an empty string still fails the regex check, but this would be consistent with the other fields and a message about the value being required might be a bit more clear.